src/zenml/orchestrators/step_launcher.py

```diff
@@ -241,10 +241,6 @@ def launch(self) -> None:
                         )
                         raise
 
-                publish_utils.update_pipeline_run_status(
```

Comment:
Just out of curiosity and for my understanding, from what I have seen from the `step_launcher` and `publish_utils` utils, the update on the pipeline run status has been completely removed from the orchestration logic. Does this have no effect on the execution of pipelines?

Reply:
Previously, after a step finished we did the following requests:
- Update the step with a new status
- Fetch all steps of the pipeline run that the step belongs to
- If necessary, update the status of the pipeline run

This now all happens server side (= in the SQLZenStore) in the first call when updating a step